GTMAppAuth/Sources/KeychainStore/KeychainStore.swift

```diff
@@ -268,6 +289,18 @@ public final class KeychainStore: NSObject, AuthSessionStore {
         .persistenceResponseString(forAuthSession: authSession) else {
       throw KeychainStore.Error.failedToCreateResponseStringFromAuthSession(authSession)
     }
+
+    if #available(macOS 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *) {
```

Comment:
This is also mixing compile-time and runtime checks

Reply:
Resolving. Let's consolidate this discussion above.